.github/workflows/golangci-lint.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Install Go 1.19.5
```

Comment:
This can be either Install Go or Install Go 1.19.6

Reply:
Adopted.